wire-library/wire-runtime-swift/build.gradle.kts

```diff
@@ -69,6 +69,24 @@ val generateSwiftProtos by tasks.creating(JavaExec::class) {
   }
 }
 
+val generateSampleProtos by tasks.creating(JavaExec::class) {
```

Comment:
There is _almost certainly_ a better way to do this, but I don't know what I'm doing in Gradle

Reply:
Why not add those files in the existing `generateTestProtos` directly?

Reply:
Added as a dependency